### DISCUSSION TITLE
Welcome to EBU6304_G058_flight Discussions!

### DISCUSSION THREAD

**Discussion by T-Atlas** [T-Atlas/EBU6304_G058_flight#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JerryZhangZZY** [comment]
Hi😁

**Reply by T-Atlas** [reply]
Hi Zeyu, thank you for your kind attention to our team project! Our team members also appreciate your repository, and we are very impressed by your excellent performance in it. 
Best wishes!
